### PR TITLE
[SPARK-27649][SS]Unify the way use 'spark.network.timeout'

### PR DESCRIPTION
**Pull request by beliefer**
## What changes were proposed in this pull request?

For historical reasons, structured streaming still has some old way of use
`spark.network.timeout`
, even though 
`org.apache.spark.internal.config.Network.NETWORK_TIMEOUT`
is now available.

## How was this patch tested?
Exists UT.

### CODE REVIEW

**Comment by beliefer** [comment]
cc @cloud-fan @zsxwing @maropu

File: external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala [hunk 1]
```diff
@@ -26,6 +26,7 @@ import org.apache.kafka.clients.consumer.ConsumerConfig
 
 import org.apache.spark.SparkEnv
 import org.apache.spark.internal.Logging
+import org.apache.spark.internal.config.Network._
```
Comment (cloud-fan) [inline external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala]: let's avoid wildcard import when unnecessary.

**Comment by srowen** [comment]
This is OK. It overlaps with https://github.com/apache/spark/pull/24270

File: external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala (hunk 1 above)
Comment (beliefer) [reply external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala]: > let's avoid wildcard import when unnecessary.

OK, I have modified.

**Comment by beliefer** [comment]
> This is OK. It overlaps with #24270

Thanks for reminding. How should I do?

**Comment by cloud-fan** [comment]
It's OK, we can rebase that PR

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Comment by beliefer** [comment]
> thanks, merging to master!

Thank you, wenchen.

**Comment by beliefer** [comment]
@cloud-fan @srowen Thanks for your help!

**Pull request closed without merging**